### DISCUSSION TITLE
new repository to help support calibrating the V400

### DISCUSSION THREAD

**Discussion by tkocou** [Guilouz/Klipper-Flsun-Speeder-Pad#141] [General]
I've put together some files which are not available in this repository. While this repository is fabulous, some of the files help speed up the calibration process.
[https://github.com/tkocou/V400-Files](url)

**Comment by tkocou** [comment]
The wiki has updates. Think of the wiki as a set of living documents which can be updated as new information becomes available.

**Reply by JackDubious** [reply]
Your link is misconfigured.
